### ISSUE TITLE
error switching from 20.0.3 to 20.0.4 singularity

### ISSUE THREAD

**Issue by saigerutherford** [poldracklab/fmriprep#2050]
fmriprep version 20.0.4 (via singularity)

When switching from fMRIPrep 20.0.3 to 20.0.4 I get the below error message. This subject completed successfully using singularity container 20.0.3 and the only thing changed in the code/data was pointing to the new singularity image for 20.0.4

```bids-validator@1.4.0

	[33m1: [WARN] The recommended file /README is missing. See Section 03 (Modality agnostic files) of the BIDS specification. (code: 101 - README_FILE_MISSING)[39m

[36m	Please visit https://neurostars.org/search?q=README_FILE_MISSING for existing conversations about this issue.[39m


        [34m[4mSummary:[24m[39m                [34m[4mAvailable Tasks:[24m[39m        [34m[4mAvailable Modalities:[24m[39m 
        24 Files, 3.52GB        task-gaze               T1w                   
        1 - Subject             task-jovi               bold                  
        1 - Session             task-rest               fieldmap              


[36m	If you have any questions, please post on https://neurostars.org/tags/bids.[39m

Making sure the input data is BIDS compliant (warnings can be ignored in most cases).
200324-14:16:49,110 nipype.workflow IMPORTANT:
	 
    Running fMRIPREP version 20.0.4:
      * BIDS dataset path: /tmp/workflow_1009/BIDS.
      * Participant list: ['1009'].
      * Run identifier: 20200324-141648_8645153e-701d-4291-99b1-db39f1124b9d.

    Spatial References: anat, fsLR:den-32k, MNI152NLin6Asym:res-2
    
200324-14:16:49,950 nipype.utils INFO:
	 No new version available.
200324-14:16:52,482 nipype.workflow IMPORTANT:
	 Creating bold processing workflow for "/tmp/workflow_1009/BIDS/sub-1009/func/sub-1009_task-gaze_run-01_bold.nii" (0.28 GB / 275 TRs). Memory resampled/largemem=1.13/1.91 GB.
200324-14:16:52,490 nipype.workflow IMPORTANT:
	 No single-band-reference found for sub-1009_task-gaze_run-01_bold.nii
200324-14:16:53,78 nipype.interface INFO:
	 We advise you to upgrade DIPY version. This upgrade will open access to more function
200324-14:16:53,82 nipype.interface INFO:
	 We advise you to upgrade DIPY version. This upgrade will open access to more function
200324-14:16:53,82 nipype.interface INFO:
	 We advise you to upgrade DIPY version. This upgrade will open access to more models
200324-14:16:53,988 nipype.workflow IMPORTANT:
	 Creating BOLD surface-sampling workflow.
Process Process-2:
Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/usr/local/miniconda/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/miniconda/lib/python3.7/site-packages/fmriprep/cli/run.py", line 693, in build_workflow
    bids_filters=bids_filters,
  File "/usr/local/miniconda/lib/python3.7/site-packages/fmriprep/workflows/base.py", line 274, in init_fmriprep_wf
    bids_filters=bids_filters,
  File "/usr/local/miniconda/lib/python3.7/site-packages/fmriprep/workflows/base.py", line 638, in init_single_subject_wf
    use_syn=use_syn,
  File "/usr/local/miniconda/lib/python3.7/site-packages/fmriprep/workflows/bold/base.py", line 946, in init_func_preproc_wf
    repetition_time=metadata['RepetitionTime'])
  File "/usr/local/miniconda/lib/python3.7/site-packages/fmriprep/workflows/bold/resampling.py", line 743, in init_bold_grayords_wf
    for hemi in 'LR'
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/interfaces/base/traits_extension.py", line 425, in validate
    value = super(MultiObject, self).validate(objekt, name, newvalue)
  File "/usr/local/miniconda/lib/python3.7/site-packages/traits/trait_types.py", line 2336, in validate
    return TraitListObject( self, object, name, value )
  File "/usr/local/miniconda/lib/python3.7/site-packages/traits/trait_handlers.py", line 2313, in __init__
    raise excp
  File "/usr/local/miniconda/lib/python3.7/site-packages/traits/trait_handlers.py", line 2305, in __init__
    value = [ validate( object, name, val ) for val in value ]
  File "/usr/local/miniconda/lib/python3.7/site-packages/traits/trait_handlers.py", line 2305, in <listcomp>
    value = [ validate( object, name, val ) for val in value ]
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/interfaces/base/traits_extension.py", line 329, in validate
    value = super(File, self).validate(objekt, name, value, return_pathlike=True)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/interfaces/base/traits_extension.py", line 134, in validate
    self.error(objekt, name, str(value))
  File "/usr/local/miniconda/lib/python3.7/site-packages/traits/trait_handlers.py", line 172, in error
    value )
traits.trait_errors.TraitError: Each element of the 'current_sphere' trait of a DynamicTraitedSpec instance must be a pathlike object or string representing an existing file, but a value of '[]' <class 'str'> was specified.
```

This is my code to run singularity:

```module load singularity

FMRIPREP=/sw/examples/singularity/fmriprep-20.0.4.simg
SURF_LICENSE=${SCRIPTDIR}/data/fs_license.txt

BIDS_DIR=/tmp/workflow_${SUB}/BIDS
OUTPUT_DIR=/tmp/workflow_${SUB}/derivatives
WORK_DIR=/tmp/workflow_${SUB}/work

singularity run --cleanenv \
$FMRIPREP $BIDS_DIR $OUTPUT_DIR participant \
--nthreads ${NTHREADS} \
--mem-mb ${MEMMB} \
--fs-license-file=$SURF_LICENSE \
--participant-label=sub-${SUB} \
-w $WORK_DIR \
--ignore slicetiming \
--cifti-output \
--use-aroma \
--aroma-melodic-dimensionality -100 \
--output-spaces anat fsLR:den-32k MNI152NLin6Asym:res-2
```

**Comment by mgxd** [comment]
since there were changes to the templateflow templates in `20.0.4`, you may need to clear your local template cache (`~/.cache/templateflow` by default) to update fetching.

**Comment by mgxd** [comment]
@saigerutherford did clearing your cache do the trick? If not, we have recently made changes to templateflow that should help avoid this down the line. You'll have to do these manually for now, but it will be shipped within the 20.1.0 release.

1. Download templateflow locally
```
pip install --upgrade templateflow==0.6.0rc2
```

2. Force the update of the templates skeleton
```
python -c "import templateflow as tf; tf.update(local=True, overwrite=False)"
```

**Comment by mgxd** [comment]
this should be automatically fixed when using 20.0.7+, reopen if you are still having issues after upgrading.
